src/hotspot/share/opto/library_call.cpp

```diff
@@ -2837,6 +2817,7 @@ bool LibraryCallKit::inline_native_classID() {
       ideal.set(result, _gvn.transform(longcon(LAST_TYPE_ID + 1)));
     } __ end_if();
 
+    Node* signaled_flag_address = makecon(TypeRawPtr::make(JfrTraceIdEpoch::signal_address()));
```

Comment:
The signal stuff can now be removed. Please also remove JfrTraceIdEpoch::signal_address() exposure.

Reply:
the path for the primitive class needs it, and I add Jfr::signal_address() to wrapper it